database/scripts/20220313_233306_ajuste_auto_down.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM movimientos_arka.formato_tipo_movimiento
+    WHERE codigo_abreviacion = 'AAT';
+
+ALTER TABLE movimientos_arka.formato_tipo_movimiento
+	DROP CONSTRAINT uq_codigo_abreviacion;
```

Comment:
Chequear primero que exista, validar si con lo siguiente es suficiente:
```suggestion
	DROP CONSTRAINT IF EXISTS uq_codigo_abreviacion;
```